scripts/verify-permissions.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/python3
+#
+# Copyright (C) 2021 Paul Keith <javelinanddart@gmail.com>
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 2 and
+# only version 2 as published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# Modified by Ilya Danilkin <nezorflame@gmail.com> @ OpenGApps
+#
+# This script helps to fix the private permission issues caused by updated APKs.
+# See https://gitlab.com/MindTheGapps/vendor_gapps/-/blob/sigma/cicd/verify-permissions.py
+# for the original file.
+
+import errno
+from glob import glob
+import os
+import subprocess
+import sys
+from xml.etree import ElementTree
+
+# Get external packages
+try:
+    from parse import parse
+except ImportError:
+    print('Please install the "parse" package via pip3.')
+    exit(errno.ENOPKG)
+try:
+    import requests
+    from requests.packages.urllib3.util import url
+except ImportError:
+    print('Please install the "requests" package via pip3.')
+    print('Make sure to have module "six" to be at least == 1.15.0')
+    exit(errno.ENOPKG)
+
+# Change working directory to the location of this script
+# This fixes relative path references when calling this script from
+# outside of the directory containing it
+os.chdir(sys.path[0])
+
+# Definitions for privileged permissions
+ANDROID_MANIFEST_XML = \
+    'https://raw.githubusercontent.com/LineageOS/android_frameworks_base/lineage-19.0/core/res/AndroidManifest.xml'
```

Comment:
Can we host our own file for this?

Reply:
And rather also a local copy in our repository than requesting a file from the internet every time we add an apk.